program/find-sum-of-numbers/FindSumOfNumbers.cs

```diff
@@ -0,0 +1,17 @@
+using System;  
+  public class SumOfDigits 
+   {  
+     public static void Main(string[] args)  
+      {  
+       int  n,sum=0,m;         
+       Console.Write("Enter a number: ");      
+       n= int.Parse(Console.ReadLine());     
+       while(n>0)      
+       {      
+        m=n%10;      
+        sum=sum+m;      
+        n=n/10;      
+       }      
```

Comment:
Hi @anandfresh , This PR is related to sum of numbers and the file contains logic related to sun of Digits.
Could you please have a look again and make necessary changes?
Thanks.